doc/src/quickstart.md

```diff
@@ -190,9 +190,9 @@ use lrlex::lrlex_mod;
 use lrpar::lrpar_mod;
 
 // Using `lrlex_mod!` brings the lexer for `calc.l` into scope.
-lrlex_mod!(calc_l);
+lrlex_mod!("calc.l");
 // Using `lrpar_mod!` brings the lexer for `calc.l` into scope.
```

Comment:
Didn't I fix this comment in one of the previous commits?

Reply:
Looks like I did: https://github.com/softdevteam/grmtools/pull/136/files#diff-cf2484e86d0b5e0ecb4d8152c4b12a8fR195
Wonder why the changes are gone here.